source/material_model/rheology/elasticity.cc

```diff
@@ -239,36 +239,93 @@ namespace aspect
       }
 
 
+      namespace
```

Comment:
I think we can update the comment and check in line 209 and onwards (elasticity.cc) now that the material averaging of the viscosity is included in the RHS term and the reaction_outputs.

Reply:
What do you think about updating these lines?

Reply:
I think it would be OK if we remove the assertion from line 213 to 224 and add an assertion that prohibits the usage of Q1 projection average. But still, when using particle-in-cell method the reaction terms for elasticity cannot be averaged, so we may have to notify the users about this somewhere.